certs_test_sm.h

```diff
@@ -0,0 +1,8400 @@
+/* certs_test_sm.h */
+/* This file was semi-manually generated using: ./gencertbuf_sm.pl */
```

Comment:
My preference is to update the existing certs_test.h as long as the SM sections are protected with SM macros. Otherwise this is just duplicated. Please update gencertbuf.pl. Any new files also need to be added to the automake include.am.